src/main/index.js

```diff
@@ -72,7 +85,7 @@ class App extends React.Component {
                         <Switch>
                             <Route exact path="/" component={Home}/>
                             <Route path="/features" component={Features}/>
-                            <Route path="/our-team" component={OurTeam}/>
+                            <Route path="/help" component={Help}/>
                             <Route path="/news" component={News}/>
                             <Route path="/contact" component={Contact}/>
                             <Route path="/log-in" component={LogIn}/>
```

Comment:
Nit: Can we agree to not hyphenate or snake case or camelcase routes so that we're always consistent? So instead of logIn or log-in or log_in we'd always just concatenate the two words directly - login?